agent/xds/envoy_versioning.go

```diff
@@ -26,16 +25,17 @@ type unsupportedVersion struct {
 }
 
 type supportedProxyFeatures struct {
-	// add version dependent feature flags here
+	// Older versions of Envoy incorrectly exploded a wildcard subscription for
+	// LDS and CDS into specific line items on incremental xDS reconnect. They
+	// would populate both InitialResourceVersions and ResourceNamesSubscribe.
+	// In this scenario they SHOULD have populated left ResourceNamesSubscribe
+	// empty (or used an explicit "*" in later versions) to imply wildcard
+	// mode. On reconnect this causes newly created listeners and clusters to
```

Comment:
```suggestion
	// LDS and CDS into specific line items on incremental xDS reconnect. They
	// would populate both InitialResourceVersions and ResourceNamesSubscribe
	// when they SHOULD have populated left ResourceNamesSubscribe empty
	// (or used an explicit "*" in later versions) to imply wildcard mode.
	// On reconnect this causes newly created listeners and clusters to
```

Reply:
Also,
> On reconnect this causes newly created listeners and clusters to be ignored

^ does `this` refer to "setting the flag true" or "the scenario above"?

Reply:
Also,

> In this scenario they SHOULD have ~populated~ left ResourceNamesSubscribe

Reply:
I reworded this.